docs/versions/unversioned/workflow/expo-cli.md

```diff
@@ -35,6 +35,7 @@ Commands:
   fetch:ios:certs [options] [project-dir]         Fetch this project's iOS certificates and provisioning profile. Writes certificates to PROJECT_DIR/PROJECT_NAME_(dist|push).p12 and prints passwords to stdout.
   fetch:android:keystore [options] [project-dir]  Fetch this project's Android keystore. Writes keystore to PROJECT_DIR/PROJECT_NAME.jks and prints passwords to stdout.
   fetch:android:hashes [options] [project-dir]    Fetch this project's Android key hashes needed to setup Google/Facebook authentication.
+  generate-module|gm [options] [project-dir]      Create new TypeScript ready module for expo based on `expo-module-template` package.
```

Comment:
I think this description needs to be improved. Right now it doesn't really explain a normal Expo CLI user what this command does.  (What's a "module for expo"?) I think it should mention universal modules. Also check the capitalization of "expo".

We should also avoid adding new short aliases like `gm`, particularly for commands that are not very frequently used.